autovalet_system/src/autovalet_system.py

```diff
@@ -61,7 +68,9 @@ def __init__(self,sim):
         self.laneCloud_topic = "/lane/pointCloud"
         self.egoLine_topic   = "/lane/egoLine"
         self.color_topic     = "/frontCamera/color/image_raw"
-        self.depth_topic     = "/depth_registered/image_rect"
+        self.depth_topic     = "/depth_registered/image_rect" if self.sim else \
```

Comment:
I see two topics here. But in one of the above files, I saw a single depth topic. ref: autovalet_lane_detection/scripts/autovalet_lane_detection/autovalet_lane_detector.py

Reply:
so I am pretty sure the topic names for sim and hw on depth are different. We keep it this way here to ensure it's subscribing to the correct topic, but for the single depth topic that you mentioned, it's fine 'cause it's not in our system code.

Reply:
Yeah. Now makes sense